common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -356,6 +356,10 @@ def get_data_for_vertical(self, vertical):
                     return (group.name, group.id)
         return (None, None)
 
+    @property
+    def tooltip_title(self):
+        return getattr(getattr(self, 'child', ''), 'tooltip_title', '')
```

Comment:
Please replace the inner `getattr(self, 'child', '')` with simply `self.child`.  The `child` member is set in the `__init__` method of this class and so will always be there.